crates/holochain_sqlite/src/sql/cell/schema/6-up.sql

```diff
@@ -0,0 +1,95 @@
+CREATE TABLE Warrant (
+  hash BLOB PRIMARY KEY ON CONFLICT IGNORE,
+  author BLOB NOT NULL,
+  timestamp INTEGER NOT NULL,
+  warrantee BLOB NOT NULL,
+  type TEXT NOT NULL,
+  blob BLOB NOT NULL
+);
+
+-- Drop foreign key constraint on action_hash in DhtOp.
+-- To achieve that, create a new table without it and drop the old table.
+CREATE TABLE IF NOT EXISTS DhtOpNew (
+  hash BLOB PRIMARY KEY ON CONFLICT IGNORE,
+  type TEXT NOT NULL,
+  basis_hash BLOB NOT NULL,
+  require_receipt INTEGER NOT NULL,  -- BOOLEAN
+  -- This is not strictly an action hash, but a foreign key to a row in the Action or Warrant table.
```

Comment:
```suggestion
  -- This is not strictly an action hash, but a reference to a row in the Action or Warrant table. Note this is not a foreign key, because a foreign key must reference a single table, and this may reference either table, depending on if the Op is a Warrant or other Op.
```

Writing this out makes it seem very muddled. I know this is supposed to be a stepping-stone refactor.